LRA TCK test failing with Java 25

When running `org.eclipse.microprofile.lra.tck.TckTests.mixedMultiLevelNestedActivity` with Java 25, the test is failing with:

```
[ERROR] org.eclipse.microprofile.lra.tck.TckTests.mixedMultiLevelNestedActivity -- Time elapsed: 1.308 s <<< ERROR!
java.util.ServiceConfigurationError: jakarta.ws.rs.client.ClientBuilder: org.jboss.resteasy.client.jaxrs.internal.ResteasyClientBuilderImpl not a subtype
	at java.base/java.util.ServiceLoader.fail(ServiceLoader.java:559)
	at java.base/java.util.ServiceLoader$LazyClassPathLookupIterator.hasNextService(ServiceLoader.java:1115)
	at java.base/java.util.ServiceLoader$LazyClassPathLookupIterator.hasNext(ServiceLoader.java:1142)
	at java.base/java.util.ServiceLoader$1.hasNext(ServiceLoader.java:1164)
	at java.base/java.util.ServiceLoader$2.hasNext(ServiceLoader.java:1246)
	at jakarta.ws.rs.client.FactoryFinder.lambda$findFirstService$1(FactoryFinder.java:178)
	at jakarta.ws.rs.client.FactoryFinder.findFirstService(FactoryFinder.java:187)
	at jakarta.ws.rs.client.FactoryFinder.find(FactoryFinder.java:114)
	at jakarta.ws.rs.client.ClientBuilder.newBuilder(ClientBuilder.java:62)
	at jakarta.ws.rs.client.ClientBuilder.newClient(ClientBuilder.java:86)
	at io.quarkus.tck.lra.NarayanaLRARecovery.waitForCallbacks(NarayanaLRARecovery.java:62)
	at org.eclipse.microprofile.lra.tck.service.LRATestService.waitForCallbacks(LRATestService.java:82)
	at org.eclipse.microprofile.lra.tck.service.LRATestService_ClientProxy.waitForCallbacks(Unknown Source)
	at org.eclipse.microprofile.lra.tck.TckTests.lambda$multiLevelNestedActivity$1(TckTests.java:615)
	at java.base/java.util.stream.ForEachOps$ForEachOp$OfInt.accept(ForEachOps.java:207)
	at java.base/java.util.stream.Streams$RangeIntSpliterator.forEachRemaining(Streams.java:104)
	at java.base/java.util.Spliterator$OfInt.forEachRemaining(Spliterator.java:710)
	at java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:570)
	at java.base/java.util.stream.ForEachOps$ForEachTask.compute(ForEachOps.java:293)
	at java.base/java.util.concurrent.CountedCompleter.exec(CountedCompleter.java:759)
	at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:511)
	at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1450)
	at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:2019)
	at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:187)
```

This needs fixing.

--- a/bom/application/pom.xml
+++ b/bom/application/pom.xml
@@ -43,7 +43,7 @@
         <microprofile-reactive-streams-operators.version>3.0.1</microprofile-reactive-streams-operators.version>
         <microprofile-rest-client.version>4.0</microprofile-rest-client.version>
         <microprofile-jwt.version>2.1</microprofile-jwt.version>
-        <microprofile-lra.version>2.0.1</microprofile-lra.version>
+        <microprofile-lra.version>2.0.2</microprofile-lra.version>
         <microprofile-openapi.version>4.1.1</microprofile-openapi.version>
         <smallrye-common.version>2.16.0</smallrye-common.version>
         <smallrye-config.version>3.16.0</smallrye-config.version>

--- a/tcks/microprofile-lra/pom.xml
+++ b/tcks/microprofile-lra/pom.xml
@@ -13,7 +13,7 @@
     <name>Quarkus - TCK - MicroProfile LRA</name>
 
     <properties>
-        <microprofile-lra-tck.version>2.0.1</microprofile-lra-tck.version>
+        <microprofile-lra-tck.version>2.0.2</microprofile-lra-tck.version>
     </properties>
 
     <build>

